notification/twilio/alertsms.go

```diff
@@ -138,26 +139,53 @@ func normalizeGSM(str string) (s string) {
 	return s
 }
 
+func renderMinGSMSegments(inputs []string, render func(inputs []string) (string, error)) (string, error) {
+	for i, s := range inputs {
+		inputs[i] = normalizeGSM(s)
+	}
+
+	size := maxGSMLen
+	for {
+		result, err := util.RenderSizeN(size, inputs, func(inputs []string) (string, error) {
+			cpy := make([]string, len(inputs))
+			for i, s := range inputs {
+				cpy[i] = strings.TrimSpace(s)
+			}
+
+			return render(cpy)
+		})
+		if errors.Is(err, util.ErrNoSolution) {
+			size += maxGSMLen
+			continue
+		}
+		if err != nil {
+			return "", err
+		}
+
+		return result, nil
+	}
+}
+
 // renderAlertMessage will render a single-segment SMS for an Alert.
 //
 // Non-GSM characters will be replaced with '?' and fields will be
 // truncated (if needed) until the output is <= maxLen characters.
```

Comment:
Can these comments be updated since `maxLen` has been changed to `appName`? It's a bit unclear now.